compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/conv_to_im2col.mlir

```diff
@@ -307,7 +307,7 @@ util.func public @conv_2d_hwcn_hwcf(%arg0: tensor<26x18x16x288xf32>, %arg1: tens
   util.return %0 : tensor<3x3x288x288xf32>
 }
 
-// CHECK-DAG:  #[[MAP:.+]] = affine_map<(d0, d1, d2, d3, d4) -> (d2, d0, d1, d4)>
+// CHECK-DAG:  #[[MAP:.+]] = affine_map<(d0, d1, d2, d3, d4) -> (d3, d0, d1, d4)>
```

Comment:
The new logic also fixes this test. d3 should be the batch dimension, and the d2 is the output channel dimension.